src/libexpr/primops.cc

```diff
@@ -2071,6 +2129,7 @@ void EvalState::createBaseEnv()
     addPrimOp("__fromJSON", 1, prim_fromJSON);
     addPrimOp("__toFile", 2, prim_toFile);
     addPrimOp("__filterSource", 2, prim_filterSource);
+    addPrimOp("__addPath", 1, prim_addPath);
```

Comment:
I'm not a fan of the name `addPath` (add to what?). Maybe it should just be `builtins.path`? That has a less imperative ring to it.